src/hermes_claude_agent_sdk/parity/v4_receipts.py

```
@@ -0,0 +1,289 @@
+
+
+def _candidate(value: Any, trial: ResultPacket) -> tuple[Candidate, str]:
+    candidate = _mapping(value, "candidate")
+    if set(candidate) != _CANDIDATE_FIELDS:
+        raise OwnershipReceiptViolation("candidate identity must contain exactly the ten v4 fields")
+    for key, length in (("plugin_sha", 40), ("host_sha", 40), ("wheel_sha256", 64), ("profile_sha256", 64)):
+        _digest(candidate[key], f"candidate.{key}", length=length)
+    expected = {"sdk_distribution": V4_SDK_DISTRIBUTION, "sdk_version": V4_SDK_VERSION,
+                "cli_version": V4_CLI_VERSION, "model": V4_MODEL, "runner_id": V4_RUNNER_ID,
+                "runner_version": V4_RUNNER_VERSION}
+    for key, value in expected.items():
```

Ruff (0.16.3)

[warning] Redefining argument with the local name `value`

(PLR1704)